src/packages.builds

```diff
@@ -53,6 +53,18 @@
     </Project>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'false' AND '$(SkipNativePackageBuild)' != 'true'" >
```

Comment:
I wouldn't expect that we set SkipNativePackageBuild == true for any of our servicing builds so I would just lump these together above.